File: C--main/DLL_Using/DLL/Class1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DLL
{
    public class hesaplama
    {
        public double ortalama(int vize1, int vize2, int final)
        {
            return ((vize1 * 0.2) + (vize2 * 0.2) + (final * 0.6));
        }
    }
}

File: C--main/Excel_SQL_DLL/ortHesapla_DLL/Class1.vb
Public Class Hesaplama
    Dim vize1 As Integer
    Dim vize2 As Integer
    Dim final As Integer

    Function OrtHesapla(vize1, vize2, final) As Double
        Dim ortalama As Double = (vize1 * 2 / 10) + (vize2 * 2 / 10) + (final * 6 / 10)
        Return ortalama
    End Function

End Class
